query/src/test/java/org/infinispan/query/api/ManualIndexingTest.java

```diff
@@ -1,58 +1,69 @@
-public class ManualIndexingTest extends MultipleCacheManagersTest {
-
-   protected static final int NUM_NODES = 4;
-   protected List<Cache<String, Car>> caches = new ArrayList<>(NUM_NODES);
-
-   @Override
-   protected void createCacheManagers() throws Throwable {
-      for (int i = 0; i < NUM_NODES; i++) {
-         EmbeddedCacheManager cacheManager = TestCacheManagerFactory.fromXml("manual-indexing-distribution.xml");
-         registerCacheManager(cacheManager);
-         Cache<String, Car> cache = cacheManager.getCache();
-         caches.add(cache);
-      }
-      waitForClusterToForm("default");
+public class ManualIndexingTest extends SingleCacheManagerTest {
+
+   // TODO HSEARCH-1807 Restore Clustered Queries
+   // protected static final int NUM_NODES = 4;
+
+   protected EmbeddedCacheManager createCacheManager() throws Exception {
+      ConfigurationBuilder c = getDefaultStandaloneCacheConfig(true);
+      c.indexing()
+            .enable()
+            .addIndexedEntity(Car.class)
+            .addProperty(SearchConfig.INDEXING_STRATEGY, SearchConfig.MANUAL)
+            .addProperty(SearchConfig.DIRECTORY_TYPE, SearchConfig.HEAP);
+      return TestCacheManagerFactory.createCacheManager(QueryTestSCI.INSTANCE, c);
    }
 
    public void testManualIndexing() {
-      caches.get(0).put("car A", new Car("ford", "blue", 400));
-      caches.get(0).put("car B", new Car("megane", "white", 300));
-      caches.get(0).put("car C", new Car("megane", "red", 500));
+      assertNumberOfCars(0, "megane");
+      assertNumberOfCars(0, "ford");
+
+      cache.put("car A", new Car("ford", "blue", 400));
+      cache.put("car B", new Car("megane", "white", 300));
+      cache.put("car C", new Car("megane", "red", 500));
 
       assertNumberOfCars(0, "megane");
       assertNumberOfCars(0, "ford");
 
       // rebuild index
-      Search.getSearchManager(caches.get(0)).getMassIndexer().start();
+      Search.getSearchManager(cache).getMassIndexer().start();
+      Search.getSearchManager(cache).refresh(Object.class);
 
       assertNumberOfCars(2, "megane");
       assertNumberOfCars(1, "ford");
    }
 
    private void assertNumberOfCars(int expectedCount, String carMake) {
-      for (Cache<?, ?> cache : caches) {
-         QueryFactory queryFactory = Search.getQueryFactory(cache);
-         Query query = queryFactory.create(String.format("FROM %s where make:'%s'", Car.class.getName(), carMake));
-         assertEquals("Expected count not met on cache " + cache, expectedCount, query.getResultSize());
-         assertEquals("Expected count not met on cache " + cache, expectedCount, query.list().size());
-      }
+      // Assert using Search6 dsl:
+      long hitCount = Search.getSearchManager(cache).search(Car.class)
+            .where(f -> f.match().field("make").matching(carMake))
+            .fetchTotalHitCount();
+      assertEquals("Expected count not met on cache " + cache, expectedCount, hitCount);
```

Comment:
It looks like you have an old version of this test, we've moved away from SearchManager and query builders. The only thing we're going to support going forward is Ickle query strings

Reply:
I added temporary to be test that alignment of the results using ISPN query and a native Search query. Going to remove it.